### ISSUE TITLE
Pretrain graphs of shapenet

### ISSUE THREAD

**Issue by Danilka234** [Ayumu-J-S/Point-JEPA#4]
Good day.
Thank you for your work! I am contacting you again:)
Could you please send me the training graphs from the settings in the config file on pretrain shapenet? I would be very grateful!

**Comment by Ayumu-J-S** [comment]
Would something like this be helpful? 
This is the training loss from the run that produced the model weights mentioned [here](https://github.com/Ayumu-J-S/Point-JEPA#:~:text=Point%2DJEPA%20pre%2Dtrained). I took the weights from the very last epoch.

![Image](https://github.com/user-attachments/assets/68d0a475-a4ac-4ac6-95d8-eb557418ccb6)

**Comment by Danilka234** [comment]
Thank you! Sorry for not answering earlier.

**Comment by jackswl** [comment]
@Ayumu-J-S great work! Do you have any reasoning/intuition on why that training loss appears that way? Why is it that there is a sharp drop, followed by an increase, and another increase until it stagnant?
